d3a_api_client/aggregator.py

```diff
@@ -32,6 +32,9 @@ def received_message(self, message):
         elif "command" in message:
             self.command_response_buffer.append(message)
 
+        if "event" in message or "command" in message:
+            self.client._on_event_or_response(message)
```

Comment:
Thanks a lot for that, i was just seeing that the logs are missing in the REST aggregator.